entrypoint.sh

```diff
@@ -19,6 +18,23 @@ HIDE_SHARES="${HIDE_SHARES:-no}"
 
 # common functions
 set_password() {
+  # check PASSWORD and PASSWORD_FILE are both not set
+  if [ ! -z "${PASSWORD}" ]
+  then
+    if [ ! -z "${PASSWORD_FILE}" ]
+    then
+      echo "ERROR: PASSSWORD and PASSWORD_FILE can not both be set. Please choose 1"
+      exit 1
+    fi
+  fi
+
+  PASSWORD="${PASSWORD:-timemachine}"
+
+  if [ ! -z "${PASSWORD_FILE}" ]
+  then
+    PASSWORD="$(cat $PASSWORD_FILE)"
```

Comment:
change `cat $PASSWORD_FILE` to `cat "${PASSWORD_FILE}"` to prevent globbing and word splitting + add `{` `}` for style preference.

Reply:
Done. Thank you